src/test/java/lotto/domain/LottoTest.java

```diff
@@ -0,0 +1,69 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+public class LottoTest {
+
+    private static Stream<Arguments> winningResultParameters() {
+        return Stream.of(
+                arguments(Arrays.asList(new Integer[]{1, 2, 3, 4, 5, 6}), Rank.FIRST),
+                arguments(Arrays.asList(new Integer[]{1, 2, 3, 4, 5, 7}), Rank.THIRD)
```

Comment:
```suggestion
                arguments(Arrays.asList(1, 2, 3, 4, 5, 6), Rank.FIRST),
                arguments(Arrays.asList(1, 2, 3, 4, 5, 7), Rank.THIRD)
```
Arrays.asList에 배열없이 바로 원하는 원소들을 넣을 수 있습니다! 
(다른 코드도 다 배열을 사용하셨네요.)